docs/installation/_index.md

```diff
@@ -21,25 +21,31 @@ aliases:
   - /getting-started/linux-install/
 ---
 
-To use Viam software with your smart machine, install and run the `viam-server` binary on the computer that you want to use to control the smart machine.
-In most cases, this will be a [single board computer (SBC)](#install-on-a-single-board-computer), like a Raspberry Pi, but you can also install `viam-server` on a [macOS or Linux computer](#install-on-a-macos-or-linux-computer).
+To use Viam software with your smart machine, install and run the `viam-server` binary on the single board computer (SBC)or other computer that is connected to your hardware.
+Installing on a computer is useful if you don't have a {{< glossary_tooltip term_id="board" text="board" >}} available, or if you want to run a Viam [service](/services/) with your robot that requires more computing resources than are otherwise available on an SBC.
```

Comment:
This is confusing because it’s using the word “computer” narrowly and sort of conflating SBC with board

Reply:
```suggestion
Installing on a laptop or desktop computer is useful if you don't have an SBC available, or if you want to run a Viam [service](/services/) with your robot that requires more computing resources than are otherwise available on an SBC.
```